dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -822,6 +822,20 @@
       # Lowering the command limit can reduce the frequency of rollback and retry.
       # This setting can also be overridden using the environment variable ZEEBE_BROKER_PROCESSING_MAXCOMMANDSINBATCH
       # maxCommandsInBatch = 100
+
+      # Allows scheduled processing tasks such as checking for timed-out jobs to run concurrently to
+      # regular processing. This is a performance optimization to ensure that processing is not interrupted by
+      # higher than usual workload for any of the scheduled tasks. This should only be disabled in case of bugs,
+      # for example if one of the scheduled tasks is not safe to run concurrently to regular processing.
```

Comment:
🔧 We could clarify that the flag should not be confused to mean that all scheduled processing tasks are run concurrently with each other. They are executed sequentially with each other and concurrently with regular processing.

Reply:
I'm not sure if that's a good idea, what if we want to change that so that scheduled tasks actually do run? IMO documenting the current behavior leaks too much implementation detail to the user.

Reply:
That's a great point! Please ignore my suggestion